React/main/src/app/screens/Game/index.js

```diff
@@ -88,20 +95,23 @@ class Game extends Component {
 
 const mapStateToProps = state => ({
   xIsNext: state.turns.xIsNext,
-  stepNumber: state.steps.stepNumber
+  stepNumber: state.steps.stepNumber,
+  winningMoves: state.winningMoves
 });
 
 const mapDispatchToProps = dispatch => ({
   toggleXIsNext: xIsNext => dispatch(toggleXIsNext(xIsNext)),
-  addStep: step => dispatch(addStep(step))
+  addStep: step => dispatch(addStep(step)),
+  movesActions: bindActionCreators(movesActions, dispatch)
```

Comment:
It is not necessary to use bindActionCreators. You can write () => dispatch(moveActions.getWinningMoves()) directly